packages/config/package.json

```diff
@@ -40,7 +40,7 @@
     "test:ci": "pnpm run test"
   },
   "dependencies": {
-    "@arktype/util": "0.0.40",
+    "@ark/util": "catalog:",
```

Comment:
TIL https://pnpm.io/catalogs

Reply:
It's a great feature! I figured I'd use it since I was updating `@ark/util` anyways, but you could migrate to this for all packages you share as non-dev dependencies across multiple packages.

Having multiple package versions can cause lots of problems, so the centralization is super valuable.